cmake/utils.cmake

```diff
@@ -467,22 +468,18 @@ print(name + version + '-' + abi + '-' + platform)"
     )
     string(REGEX REPLACE "-" "_" wheel_name ${GEODE_WHEEL_NAME})
     set(wheel_file "${wheel_output_path}/dist/${wheel_name}-${WHEEL_VERSION}-${wheel_sufix}.whl")
-    if(WIN32 AND NOT ${GEODE_WHEEL_SUPERBUILD})
-        add_custom_target(wheel
-            COMMAND ${CMAKE_COMMAND} -E copy_directory "${wheel_build_directory}/${binary_folder}/$<CONFIG>" "${wheel_output_directory}/${binary_folder}"
-            COMMAND ${CMAKE_COMMAND} -E copy_directory "${wheel_build_directory}/share" "${wheel_output_directory}/share"
-            COMMAND ${PYTHON_EXECUTABLE} setup.py bdist_wheel
-            WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/wheel
-        )
+    if(${GEODE_WHEEL_SUPERBUILD})
+        set(wheel_config_folder "")
     else()
-        add_custom_target(wheel
-            COMMAND ${CMAKE_COMMAND} -E copy_directory "${wheel_build_directory}/${binary_folder}" "${wheel_output_directory}/${binary_folder}"
-            COMMAND ${CMAKE_COMMAND} -E copy_directory "${wheel_build_directory}/share" "${wheel_output_directory}/share"
-            COMMAND ${CMAKE_COMMAND} -E remove "${wheel_output_directory}/${binary_folder}/*.py"
-            COMMAND ${PYTHON_EXECUTABLE} setup.py bdist_wheel
-            WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/wheel
-        )
+        set(wheel_config_folder "$<$<BOOL:${isMultiConfig}>:$<CONFIG>>")
     endif()
+    add_custom_target(wheel
+        COMMAND ${CMAKE_COMMAND} -E copy_directory "${wheel_build_directory}/${binary_folder}/${wheel_config_folder}" "${wheel_output_directory}/${binary_folder}"
```

Comment:
Are you sure you won't have a // instead of / if wheel_config_folder="" ? Will it be a problem if you do ?

Reply:
// is not an issue